sass/components/_breadcrumb.scss

```diff
@@ -130,6 +130,7 @@ a.breadcrumb__link {
 @include breakpoint(screen-md-min) {
   .breadcrumb {
     margin: 0;
```

Comment:
Instead of having two lines for margin rule, let's merge and change to `margin: 0 0 -1px 0;`